wordpress-mobile/WordPress-Android
# Issue/9815 domain suggestion header

Fixes #9815 

All the work in this PR was done in #9865
PR got an OK from the Design Review.

I'm submitting it again here since we updated to android-x, and I had to removed failed import from the test.

To test:
- Install the wasabi build.
- Switch to a site with unclaimed domain credit.
- Tap on Register Domain CTA prompt.
- Notice the header above the domain suggestion screen that disappears when you type a search query and appears when you remove it.

[![Image from Gyazo](https://i.gyazo.com/3fc702cced23012d5556e3ffa35050f7.gif)](https://gyazo.com/3fc702cced23012d5556e3ffa35050f7)
